components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilderTest.java

```diff
@@ -75,18 +81,28 @@
 public class DefaultIDTokenBuilderTest extends IdentityBaseTest {
 
     public static final String TEST_APPLICATION_NAME = "DefaultIDTokenBuilderTest";
+    private static final String AUTHORIZATION_CODE = "AuthorizationCode";
+    private static final String AUTHORIZATION_CODE_VALUE = "55fe926f-3b43-3681-aecc-dc3ed7938325";
     private DefaultIDTokenBuilder defaultIDTokenBuilder;
     private OAuth2AccessTokenReqDTO tokenReqDTO = new OAuth2AccessTokenReqDTO();
     private OAuthTokenReqMessageContext messageContext = new OAuthTokenReqMessageContext(tokenReqDTO);
     private OAuth2AccessTokenRespDTO tokenRespDTO = new OAuth2AccessTokenRespDTO();
+    private OAuth2AuthorizeReqDTO authzTokenReqDTO = new OAuth2AuthorizeReqDTO();
+    private OAuthAuthzReqMessageContext authzMessageContext = new OAuthAuthzReqMessageContext(authzTokenReqDTO);
+    private OAuth2AuthorizeRespDTO authzTokenRespDTO = new OAuth2AuthorizeRespDTO();
```

Comment:
These look like objects that should be local to a test.
Can we consider instantiating them with the tests?